test/addressSpec.js

```diff
@@ -0,0 +1,146 @@
+var addr = require('../src/js/address');
+
+describe('address functions', function() {
+  describe('is adddress duplicate', function() {
+    var address,
+      duplicates;
+
+    beforeEach(function(){
+      address = '123 Main St., Somecity, SomeWhere, 12345';
```

Comment:
It does change in the [`should NOT be duplicate - address NOT a string` test](https://github.com/cfpb/rural-or-underserved-test/pull/86/files#diff-86553dd34e870098d99efd5da528fbb9R23).